product_set/views/product_set.xml

```diff
@@ -50,7 +50,7 @@
                                 <field name="sequence" widget="handle" />
                                 <field
                                     name="product_id"
-                                    attrs="{'required': [('display_type', '=', False)]}"
+                                    required="display_type == 'False'"
```

Comment:
I think display_type is not "False" but False, or? Therefore I think this should be correct.
`required="not display_type"`

Reply:
yeah right